code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -29,16 +29,20 @@
 	var/trit = . ? .[MOLES] : 0
 	. = air_gases[/datum/gas/hydrogen]
 	var/h2 = . ? .[MOLES] : 0
+	. = air_gases[/datum/gas/antinoblium]
+	var/an = . ? .[MOLES] : 0
+	. = air_gases[/datum/gas/nitrogen]
+	var/n2 = . ? .[MOLES] : 0
 	if(active_hotspot)
 		if(soh)
-			if(plas > 0.5 || trit > 0.5 || h2 > 0.5)
+			if(plas > 0.5 || trit > 0.5 || h2 > 0.5 || (an > 0.5 && n2 > 0.5))
```

Comment:
@LemonInTheDark this is why i wanted to generalize hotspots

Reply:
I was also thinking about that. We have cache_results for fire, so maybe we could base it on that instead? Wouldn't have to get a bunch of gases either.